src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--big"
+      data-qa="big"
+    >
+      <label class="search-form__label search-form__label--big">
+        <input
+          data-qa="keypress"
+          class="search-form__input search-form__input--big"
+          type="text"
+          placeholder="Try “Los Angeles“"
+        />
+      </label>
+    </form>
 
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-form search-form--small"
+      data-qa="small"
+    >
+      <label class="search-form__label search-form__label--small">
+        <input
+          data-qa="hover"
+          class="search-form__input search-form__input--small"
+          type="text"
+          placeholder="Try “Los Angeles“"
+        />
```

Comment:
The attributes of the <input> element should start each on a new line with 2-space indentation related to the tag. The closing bracket of the <input> tag should be aligned with the opening bracket.